api/src/pcapi/static/backoffice/js/addons/pc-textarea-counter.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Displays the number of characters in a textarea.
+ * @example
+ * <textarea class="pc-textarea-counter" id="textarea-id"></textarea>
```

Comment:
```suggestion
 * <textarea class="pc-textarea-counter"></textarea>
```
L'id ne sert plus ;)

Reply:
indeed je retire 🙏